<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeypadDlg</class>
 <widget class="QDialog" name="KeypadDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>410</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>312</width>
    <height>410</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>312</width>
    <height>410</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Keypad</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#KeypadDlg {
/*background-color: rgb(237, 237, 237);*/


	background-color: rgb(247, 248, 249);
/*	background-color: rgb(196, 202, 209);*/
}
QPushButton {
	border-radius:0px;
/*

	border: 0px solid rgba(255,255,255,100);
	background-color:rgb(255,255,255,100);
*/
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_top" native="true">
     <property name="styleSheet">
      <string notr="true">/*
#widget_top {
	background-color: rgb(247, 248, 249);
}
*/</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>15</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>15</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QLabel" name="label_minMax">
           <property name="text">
            <string notr="true">(min ~ max)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>278</width>
          <height>44</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>44</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_input" native="true">
           <property name="minimumSize">
            <size>
             <width>278</width>
             <height>44</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>278</width>
             <height>44</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#widget_input {
	background-image: url(:/global/images/keypad/input.png);
	border: none;
}
</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="topMargin">
             <number>1</number>
            </property>
            <property name="bottomMargin">
             <number>1</number>
            </property>
            <item>
             <widget class="QLineEdit" name="lineEdit">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgba(255, 255, 255, 0);
border: none</string>
              </property>
              <property name="maxLength">
               <number>256</number>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_middle" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_decrease">
           <property name="minimumSize">
            <size>
             <width>138</width>
             <height>42</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>138</width>
             <height>42</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	background-image: url(:/global/images/keypad/valuedown.png);
	border: none;
}
QPushButton:pressed {
	background-image: url(:/global/images/keypad/valuedown_pressed.png);
}
</string>
           </property>
           <property name="iconSize">
            <size>
             <width>118</width>
             <height>42</height>
            </size>
           </property>
           <property name="autoRepeat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_increase">
           <property name="minimumSize">
            <size>
             <width>138</width>
             <height>42</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>138</width>
             <height>42</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	background-image: url(:/global/images/keypad/valueup.png);
	border: none;
}
QPushButton:pressed {
	background-image: url(:/global/images/keypad/valueup_pressed.png);
}
</string>
           </property>
           <property name="iconSize">
            <size>
             <width>118</width>
             <height>42</height>
            </size>
           </property>
           <property name="autoRepeat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_button" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">/*
#widget_button {
	background-color: rgb(247, 248, 249);
}
*/</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <property name="horizontalSpacing">
       <number>8</number>
      </property>
      <property name="verticalSpacing">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>61</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-image: url(:/global/images/keypad/7.png);
	border: none;
}
QPushButton:pressed {
	background-image: url(:/global/images/keypad/7_pressed.png);
}
</string>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>61</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>61</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-image: url(:/global/images/keypad/8.png);
	border: none;
}
QPushButton:pressed {
	background-image: url(:/global/images/keypad/8_pressed.png);
}
</string>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>61</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QPushButton" name="pushButton_9">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>61</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-image: url(:/global/images/keypad/9.png);
	border: none;
}
QPushButton:pressed {
	background-image: url(:/global/images/keypad/9_pressed.png);
}
</string>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>61</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="pushButton_back">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>79</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>79</width>
          <height>60</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-image: url(:/global/images/keypad/delete.png);
	border: none;
}
QPushButton:pressed {
	background-image: url(:/global/images/keypad/delete_pressed.png);
}
</string>
        </property>
        <property name="iconSize">
         <size>
          <width>79</width>
          <height>60</height>
         </size>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>61</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-image: url(:/global/images/keypad/4.png);
	border: none;
}
QPushButton:pressed {
	background-image: url(:/global/images/keypad/4_pressed.png);
}
</string>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>61</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>61</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-image: url(:/global/images/keypad/5.png);
	border: none;
}
QPushButton:pressed {
	background-image: url(:/global/images/keypad/5_pressed.png);
}
</string>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>61</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="2">
       <widget class="QPushButton" name="pushButton_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>61</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-image: url(:/global/images/keypad/6.png);
	border: none;
}
QPushButton:pressed {
	background-image: url(:/global/images/keypad/6_pressed.png);
}
</string>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>61</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="pushButton_reset">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>79</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>79</width>
          <height>60</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-image: url(:/global/images/keypad/reset.png);
	border: none;
}
QPushButton:pressed {
	background-image: url(:/global/images/keypad/reset_pressed.png);
}
</string>
        </property>
        <property name="iconSize">
         <size>
          <width>79</width>
          <height>60</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>61</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-image: url(:/global/images/keypad/1.png);
	border: none;
}
QPushButton:pressed {
	background-image: url(:/global/images/keypad/1_pressed.png);
}
</string>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>61</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>61</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-image: url(:/global/images/keypad/2.png);
	border: none;
}
QPushButton:pressed {
	background-image: url(:/global/images/keypad/2_pressed.png);
}
</string>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>61</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QPushButton" name="pushButton_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>61</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-image: url(:/global/images/keypad/3.png);
	border: none;
}
QPushButton:pressed {
	background-image: url(:/global/images/keypad/3_pressed.png);
}
</string>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>61</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="4" rowspan="2">
       <widget class="QPushButton" name="pushButton_enter">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>79</width>
          <height>127</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>79</width>
          <height>127</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-image: url(:/global/images/keypad/enter.png);
	border: none;
}
QPushButton:pressed {
	background-image: url(:/global/images/keypad/enter_pressed.png);
}
</string>
        </property>
        <property name="iconSize">
         <size>
          <width>79</width>
          <height>127</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pushButton_plusMinus">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>61</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-image: url(:/global/images/keypad/plusminus.png);
	border: none;
}
QPushButton:pressed {
	background-image: url(:/global/images/keypad/plusminus_pressed.png);
}
</string>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>61</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="pushButton_0">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>61</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-image: url(:/global/images/keypad/0.png);
	border: none;
}
QPushButton:pressed {
	background-image: url(:/global/images/keypad/0_pressed.png);
}
</string>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>61</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="2">
       <widget class="QPushButton" name="pushButton_point">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>61</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-image: url(:/global/images/keypad/dot.png);
	border: none;
}
QPushButton:pressed {
	background-image: url(:/global/images/keypad/dot_pressed.png);
}
</string>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>61</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
